packages/sanity/src/core/studio/addonDataset/types.ts

```diff
@@ -0,0 +1,17 @@
+import {type SanityClient} from '@sanity/client'
+
+/**
+ * @beta
+ * @hidden
+ */
+export interface AddonDatasetContextValue {
+  /**
+   * Addon dataset client, currently called `comments` dataset.
+   */
+  client: SanityClient | null
+  isRunningSetup: boolean
```

Comment:
Since we renamed `runSetup` to `createAddonDasaset` – should we also rename `isRunningSetup` to `isCreatingDataset`? 😇

Reply:
sure!!